src/openrct2/actions/SurfaceSetStyleAction.hpp

```diff
@@ -114,17 +114,17 @@ DEFINE_GAME_ACTION(SurfaceSetStyleAction, GAME_COMMAND_CHANGE_SURFACE_STYLE, Gam
 
         money32 surfaceCost = 0;
         money32 edgeCost = 0;
-        for (int32_t x = validRange.GetLeft(); x <= validRange.GetRight(); x += 32)
+        for (CoordsXY coords = { validRange.GetLeft(), validRange.GetTop() }; coords.x <= validRange.GetRight(); coords.x += 32)
         {
-            for (int32_t y = validRange.GetTop(); y <= validRange.GetBottom(); y += 32)
+            for (coords.y = validRange.GetTop(); coords.y <= validRange.GetBottom(); coords.y += 32)
```

Comment:
Could you replace the `+= 32` on the lines you touched with `+= COORDS_XY_STEP`?

Reply:
It is not merged yet, so I can't.